__init__.py

```diff
@@ -1,9 +0,0 @@
-# encoding: utf-8
-
-# this is a namespace package
-try:
-    import pkg_resources
-    pkg_resources.declare_namespace(__name__)
-except ImportError:
-    import pkgutil
-    __path__ = pkgutil.extend_path(__path__, __name__)
```

Comment:
I can't remember why this was put here.  Which makes me a little worried about removing it without clear rationale.  Do you understand what this file is doing and why it might be here?

Reply:
I will go through the code to understand why removing the code works.